tools/integration_tests/local_file/helpers/gcs_client.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2023 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helpers
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"path"
+	"strings"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/setup"
+)
+
+func setBucketAndObjectBasedOnTypeOfMount(bucket, object *string) {
+	*bucket = setup.TestBucket()
+	if setup.DynamicBucketMounted() != "" {
+		*bucket = setup.DynamicBucketMounted()
+	}
+	if setup.OnlyDirMounted() != "" {
+		*object = path.Join(setup.OnlyDirMounted(), *object)
+	}
+}
+
+// ReadObjectFromGCS downloads the object from GCS and returns the data.
+func ReadObjectFromGCS(object string) (string, error) {
```

Comment:
Can we use [this](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/tools/integration_tests/util/operations/file_operations.go#L392) to download file from gcs and after that read file content and return?

Reply:
The gsutil CLI was closing the file handles and syncing the local files to GCS, hence using storage client.